### PR TITLE
ts/DEMA-indicator

### PR DESCRIPTION
**Pull request by karolkolodziej**

### CODE REVIEW

File: ts/Stock/Indicators/DEMA/DEMAIndicator.ts [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface DEMAIndicatorLinkedParentSeries extends LineSeries {
+            EMApercent?: number;
+        }
+    }
+}
```
Comment (bre1470) [inline ts/Stock/Indicators/DEMA/DEMAIndicator.ts:43]: This should be deleted. Replace the DEMAIndicatorLinkedParentSeries type with `LineSeries` and add in this place:
```ts
/* *
 *
 *  Declarations
 *
 * */

declare module '../../../Core/Series/SeriesLike' {
    interface SeriesLike {
        EMApercent?: number;
    }
}
```

File: ts/Stock/Indicators/DEMA/DEMAIndicator.ts [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+     *               pointPlacement, pointRange, pointStart, showInNavigator,
+     *               stacking
+     * @requires     stock/indicators/indicators
+     * @requires     stock/indicators/ema
+     * @requires     stock/indicators/dema
+     * @optionparent plotOptions.dema
+     */
+    public static defaultOptions: DEMAOptions = merge(EMAIndicator.defaultOptions, {
+
+    } as DEMAOptions)
```
Comment (bre1470) [inline ts/Stock/Indicators/DEMA/DEMAIndicator.ts:77]: If there were not default options, you can just assign `merge(EMAIndicator.defaultOptions);`

**Review by bre1470 — CHANGES_REQUESTED** [review]
Thank you! See the comments for requested changes.

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by TorsteinHonsi — APPROVED** [review]

**Comment by TorsteinHonsi** [comment]
Is it ready to merge? If so, please mark it "Ready for review".

**Pull request merged**